### PR TITLE
use results model for wordcloud

### PR DESCRIPTION
**Pull request by lukavdplas**
Refactors the `WordCloudComponent` so it uses the `Results` class (part of #1468)

### CODE REVIEW

File: frontend/src/app/services/api.service.ts [hunk 1]
```diff
@@ -139,9 +139,9 @@ export class ApiService {
     }
 
     // Visualization
-    public wordCloud(data: WordcloudParameters): Promise<AggregateResult[]> {
+    public wordCloud(data: WordcloudParameters): Observable<AggregateResult[]> {
```
Comment (JeltevanBoheemen) [inline frontend/src/app/services/api.service.ts:142]: Let's make this a habit ;)

File: frontend/src/app/visualization/visualization.component.html [hunk 2]
```diff
@@ -105,8 +105,8 @@
                         (error)="setErrorMessage($event)"
                         (totalTokenCountAvailable)="showTokenCountOption = $event"></ia-histogram>
                 </ng-container>
-                <ia-wordcloud *ngSwitchCase=" 'wordcloud' " [corpus]="corpus" [queryModel]="queryModel" [visualizedField]="visualizedField"
-                    [resultsCount]="resultsCount" [asTable]="freqtable" [palette]="palette"
+                <ia-wordcloud *ngSwitchCase=" 'wordcloud' " [queryModel]="queryModel"
```
Comment (JeltevanBoheemen) [inline frontend/src/app/visualization/visualization.component.html:108]: What's with the whitespace around `'wordcloud'`? Is that interpreted? Does it break if we remove it?

**Review by JeltevanBoheemen — APPROVED** [review]
This makes total sense. Small questions, but good job consider approved!

File: frontend/src/app/visualization/visualization.component.html (hunk 2 above)
Comment (lukavdplas) [reply frontend/src/app/visualization/visualization.component.html:108]: Knowing I-analyzer, this is either strange formatting that was just never changed, or removing the space will somehow break all CSV downloads.

Probably the former.

**Pull request merged**